### DISCUSSION TITLE
Storybook without create react app

### DISCUSSION THREAD

**Discussion by wildeyes** [storybookjs/storybook#15722] [General]
Hello, thank you for this package!

Is there a way to setup storybook on a typescript / react library, without create-react-app?

**Comment by jonniebigodes** [comment]
@wildeyes indeed it is. It depends on how you're willing to set up your development workflow. If you want to build it by yourself see [here](https://www.romerorocha.com/posts/react-ts-rollup-lib/) for a detailed walkthrough on how you could set up a component library with TypeScript, React, and Storybook. If you want to use tooling, you could probably take a look at [TSDX](https://tsdx.io/), it includes a template prebuilt with Storybook.


Stay safe
